core/src/main/java/org/apache/servicecomb/core/provider/producer/ProducerBootListener.java

```diff
@@ -63,6 +76,9 @@ public void onAfterTransport(BootEvent event) {
       Swagger swagger = schemaMeta.getSwagger();
       swagger.addScheme(Scheme.forValue(swaggerSchema));
       String content = SwaggerUtils.swaggerToString(swagger);
+      if (exportToFile) {
```

Comment:
```
      if (exportToFile) {
        exportToFile(String.format(filePath, microservice.getServiceName(), schemaMeta.getSchemaId()), content);
      } else {
      LOGGER.info("generate swagger for {}/{}/{}, swagger: {}",
          microserviceMeta.getAppId(),
          microserviceMeta.getMicroserviceName(),
          schemaMeta.getSchemaId(),
          content);
}
```

is better? Do you think so? By default not print contents in logs, because users can read swagger in temporary file or in service center.